src/main/java/com/synopsys/blackduck/upload/client/uploaders/BinaryUploader.java

```diff
@@ -62,15 +64,16 @@ public class BinaryUploader extends AbstractUploader<BinaryUploadStatus> {
      */
     @Override
     protected BodyContent createBodyContent(Path filePath) {
-        return new MultipartBodyContent(
-            Map.of("fileupload", filePath.toFile()),
-            Map.of(
-                "projectName", binaryScanRequestData.getProjectName(),
-                "version", binaryScanRequestData.getVersion(),
-                "codeLocationName", binaryScanRequestData.getCodeLocationName().orElse(""),
-                "codeLocationUri", binaryScanRequestData.getCodeLocationUri().orElse("")
-            )
-        );
+        final Map<String, File> contentMap = new HashMap<String, File>() {{
+            put("fileupload", filePath.toFile());
+        }};
+        final Map<String, String> metaDataMap = new HashMap<String, String>() {{
+            put("projectName", binaryScanRequestData.getProjectName());
+            put("version", binaryScanRequestData.getVersion());
+            put("codeLocationName", binaryScanRequestData.getCodeLocationName().orElse(""));
+            put("codeLocationUri", binaryScanRequestData.getCodeLocationUri().orElse(""));
+        }};
```

Comment:
The instance initializers appear to be coming up as a code smell in sonar. Can we able to split initializing the Maps and a putting the key/values?

Reply:
Those 'code smells' are silly.

Sure.

Reply:
Done